docs/installpackage.md

```diff
@@ -0,0 +1,59 @@
+# Installing Our Package
+
+Our package name is currently `EducationExtension`.
+
+Our top level namespace is currently `Extension`.
+
+Our GPR source is `https://nuget.pkg.github.com/dotnet-interactive-learning/index.json`.
```

Comment:
It's usually useful to avoid initialisms unless they're very, very commonly known.